docs/src/main/tut/TypedML.md

```diff
@@ -15,17 +29,226 @@ spark.sparkContext.setLogLevel("WARN")
 
 import spark.implicits._
 ```
+
+## Example 1: predict a continuous value using a `TypedRandomForestRegressor`
+
+In this example, we want to predict `field3` of type Double from `field1` and `field2` using 
+a `TypedRandomForestRegressor`.
+
+### Training
+
+As with Spark ML API, we use a `TypedVectorAssembler` (the type-safe equivalent of `VectorAssembler`) to compute feature vectors:
+
+```tut:silent
+import frameless._
+import frameless.syntax._
+import frameless.ml._
+import frameless.ml.feature._
+import frameless.ml.regression._
+import org.apache.spark.ml.linalg.Vector
+```
+
+```tut:book
+case class Data(field1: Double, field2: Int, field3: Double)
```

Comment:
Maybe we should rename field3 to "label" and field1 and field2 to feature1, feature2? By giving them a semantic naming it will help the reader to follow the example easier.

Reply:
By using generic field names like field1, field2 and field3, I wanted to emphasize the fact that TypedTransformers and TypedEstimators did not rely on hard-coded field names for features and label, and can operate on already-well-named user data fields like "temperature", "housePrice", ... 
What do you think?

Reply:
I like it!